drivers/bridge/bridge.go

```diff
@@ -147,6 +147,27 @@ func (c *networkConfiguration) Validate() error {
 	return nil
 }
 
+func (c *networkConfiguration) copyDefaultConfigs(option options.Generic, config *networkConfiguration) {
```

Comment:
Just realized that we cannot make this a generic fix. Especially because genericData has overloaded meaning in different contexts.
This fix will apply properly (which is the most common case), where genericData is a map[string]interface{}. Hence moving the copyFromDefaultConfigs for that particular case.